lib/iris/experimental/regrid_conservative.py

```diff
@@ -76,6 +72,9 @@ def _make_esmpy_field(x_coord, y_coord, ref_name='field',
         Add a mask item to the grid, assigning it 0/1 where mask=False/True.
 
     """
+    # Lazy import so we can build the docs with no ESMF.
```

Comment:
Perhaps the comment should be "Lazily import ESMF, as it is only an optional dependency.".